dipy/data/__init__.py

```diff
@@ -261,7 +261,7 @@ def mrtrix_spherical_functions():
     -------
     func_coef : array (2, 3, 4, 45)
         Functions represented by the coefficients associated with the
-        mrtrix spherical harmonic basis of order 8.
+        mrtrix spherical harmonic basis of order (l) 8.
```

Comment:
Should it be "basis of maximal order"? Following the change from `sh_order` to `sh_order_max`.